pull_request_template.md

```diff
@@ -1,21 +1,11 @@
 ## What
-<!-- Add a list of features/ bugs/ ... that this PR handles -->
 * 
-*
 
 ## How
-<!-- 
-List all the context the reviewer needs
```

Comment:
maybe we can keep these in but put them above the titles, like JSDoc (and maybe a little shorter) so you can start typing under the title but still have the explanation

Reply:
fixed!